.github/workflows/optimize-backend-linting.yml

```diff
@@ -2,14 +2,18 @@ name: Optimize Backend Linting
 on:
   push:
     branches:
-      - master
-    paths-ignore:
-      - "optimize/client/**"
-      - ".ci/**"
+      - main
+      - maintenance/**
```

Comment:
in the mono repo, I believe we use `stable` branches rather than `maintenance`-prefixed branches. I have created an issue for @misiekhardcore to look into this: https://github.com/camunda/camunda-optimize/issues/13938

Perhaps already changing this here makes sense

Reply:
I think it does

Reply:
I renamed the `maintenance` to `stable` in this PR and there is [another one](https://github.com/camunda/camunda/pull/20656) to rename all the remaining references to `maintenance` or `master`

Reply:
@misiekhardcore - Please see my comment on the other PR, it is relevant here too

Reply:
I replied there, see [here](https://github.com/camunda/camunda/pull/20656#issuecomment-2262190890). Also @cmur2 I think it would be good if you read that conversation

Reply:
I read it, thanks :+1: